stock_inventory_discrepancy/security/stock_inventory_discrepancy_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo noupdate="0">
+
+    <record id="group_stock_control_manager" model="res.groups">
+        <field name="name">Control Manager</field>
+        <field name="category_id" ref="base.module_category_warehouse_management"/>
+        <field name="implied_ids"
```

Comment:
Okay, in that case I would rename the group, "Inventory Control Right" or something similar? what do you think?